geode-redis/src/test/java/org/apache/geode/redis/internal/executor/general/PexpireExecutorJUnitTest.java

```diff
@@ -0,0 +1,33 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more contributor license
+ * agreements. See the NOTICE file distributed with this work for additional information regarding
+ * copyright ownership. The ASF licenses this file to You under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance with the License. You may obtain a
+ * copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License
+ * is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express
+ * or implied. See the License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+package org.apache.geode.redis.internal.executor.general;
+
+import static org.assertj.core.api.AssertionsForInterfaceTypes.assertThat;
+
+import org.junit.Test;
+
+import org.apache.geode.redis.internal.Executor;
+import org.apache.geode.redis.internal.executor.ExpireExecutor;
+import org.apache.geode.redis.internal.executor.PExpireExecutor;
+
+public class PexpireExecutorJUnitTest {
+
+  @Test
+  public void isAnInstanceOfExpireClass() {
+    Executor subject = new PExpireExecutor();
```

Comment:
I'm sure this test is important, but it seems odd that there's nothing else to test other than it is an `instanceof`

Reply:
Not sure how to make this more clear- this idea is that there is a base class (expireExecutor) where 99% of functionality happens.   The Pexpireexecutor class calls uses the default base class for all functionality except for converting the seconds paramater to milliseconds.  SO we tests the millisecond functionality in the  extended integration test for the pexpire class, and test that it is extending its parent.  I was a little unsure about this one, but it didn;t seem that it was worth having all of the other unit tests to test the underlying base class (again).  Suggestions welcome...

Reply:
Oh the test is fine! I meant it was just weird to me that there wasn't anything else to test.  Sorry for the confusion!  Thank you for the clarifying response explaining that all the cool stuff happens in the base class